fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PrefixingIdentifierTranslator.java

```diff
@@ -39,7 +39,7 @@
 /**
  * A very simple {@link IdentifierConverter} which translates JCR paths into Fedora subjects with
  * a configurable resource namespace (e.g., a baseURL).  When a REST API context is available for
- * constructing URIs, {@link HttpResourceConverter} should be used instead.
+ * constructing URIs, org.fcrepo.http.commons.api.rdf.HttpResourceConverter should be used instead.
```

Comment:
I wanted the same thing, but `fcrepo-kernel-impl` does not have dependency visibility into `fcrepo-http-commons`.
Hence the @link syntax does not work.